src/process/redux/store.js

```diff
@@ -21,14 +22,16 @@ const reducer = combineReducers({
   user: userReducer,
   events: eventsReducer,
   drivers: driversReducer,
+  transient: transientReducer,
```

Comment:
As I told you on our daily standup, we must black list this reducer so on every refresh of the page, or when we close that tab, we have to delete the transient state, not PERSIST it.

Reply:
Also make the create logger in one single line.

Reply:
done